managed-identity-mysql/README.md

```diff
@@ -0,0 +1,59 @@
+# Access Azure Database for MySQL with managed identity
+
+This sample shows how to access Azure Database for MySQL with managed identity in `Azure Spring Cloud`.
+
+## Prerequisite
+
+* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Portal](https://docs.microsoft.com/en-us/azure/key-vault/secrets/quick-create-portal) or [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/keyvault?view=azure-cli-latest#az-keyvault-create)
+* An existing Azure Database for MySQL instance with a database with name `demo`. If you need to create Azure Database for MySQL, you can use the [Azure Portal](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal) or [Azure CLI](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-cli)
+
+## How to run
+
+1. Run mvn clean package after specifying the information of your Key Vault URI and Azure Database for MySQL in application.properties.

```

Comment:
Can we be more specific about below tasks?
- How to get the Key Vault endpoint
- How to get the URL and username of MySQL
- Where is the `application.properties` file to update

Reply:
Also let's put `mvn clean package` as the 2nd step so that people don't miss is accidentally.